seqr/views/apis/data_manager_api.py

```diff
@@ -458,6 +460,32 @@ def load_phenotype_prioritization_data(request):
     })
 
 
+@data_manager_required
+def write_pedigree(request, project_guid):
+    project = Project.objects.get(guid=project_guid)
+
+    possible_file_paths = [
+        f'gs://seqr-datasets/v02/{project.get_genome_version_display()}/RDG_{sample_type}_Broad_{callset}/base/projects/{project.guid}'
```

Comment:
You may want to define a constant for the `gs://seqr-datasets/v02` and share it among the seqr.